code/cavity2d/Makefile

```diff
@@ -0,0 +1,12 @@
+# Default Target
+.PHONY: all
+all: CAVITY_2D_STAGGERED
+
+# CAVITY_2D_STAGGERED
```

Comment:
This comment is unnecessary.  The next line is crystal clear for the target.  Please remove this line.

Reply:
# Default Target

CAVITY_2D_STAGGERED: clean
	gfortran -O3 CAVITY_2D_STAGGERED.f90

.PHONY: clean
clean:
	rm -f CAVITY_2D_STAGGERED

Reply:
I am not sure I understand your comment?